### ISSUE TITLE
DAU secretariat should be able categorize a post as a cheap fake or a deep fake or additional classifications.

### ISSUE THREAD

**Issue by tarunima** [tattle-made/DAU#48]
The MCA is deciding on definitions. The DAU secretariat should be able to classify the posts as per the decided definitions.

This feature is needed by launch. The definitions however might only be available by the 21st.

**Comment by areeba-dau** [comment]
Added 14 new tags in the rows 14 to 27 in https://docs.google.com/spreadsheets/d/1YLS4MyCmGuvMt95rhAeqgZ9hHor5HM8fZ_KqdtNY-F8/edit#gid=321819565 , sub-sheet "Tags-1"

**Comment by dennyabrain** [comment]
some notes : the tags facebook and instagram already existed so I havent added them again. took a call to add EVM.

**Comment by dennyabrain** [comment]
@areeba-dau this change is now live. Did a cursory test. Feel free to report any bugs here.

**Comment by areeba-dau** [comment]
Generally, when I type a single letter, all tags starting with that letter show up but this doesn't happen every time. After adding one tag I have to refresh the page to be able to see the list pop-up again when I type the letters for the second tag.

**Comment by dennyabrain** [comment]
@aatmanvaidya @duggalsu can you check if you all are able to recreate this with your user IDs?
Steps to recreate : 
1. Login with your credentials on the production instance
2. Select any query and click on its associated "add verification notes" button
3. In the tags field, try typing a word and see if you can select an associated tag.

Note : the tags arent saved in database till you hit the "Save" button. Hence feel free to test this but DO NOT press the save button. If you do, let us know asap so we can remove it from backend.


@areeba-dau your point number (2) gave me a hunch of what might be happening. I tried adding INC by typing "I"( capital I) and nothing came up but when i type "i" (small i), INC and other tags starting with I came up. 
So our code doesnt seem to be handling capital letters correctly. Could you try on your end and confirm if my assessment is correct. Can you also see if this explains your point (1)? Maybe its possible that you were getting inconsistent behaviour because you were typing capital letters without realizing.
Its a bug we'll fix. Users should be allowed to type the word however.

**Comment by dennyabrain** [comment]
@aatmanvaidya the fix might be to modify this line 
https://github.com/tattle-made/dau-dashboard/blob/69d5381735dff51fae0dc60f6822c9f1e9867845/assets/js/tag-selector-hooks.js#L189

It should be changed from  `tag.toLowerCase().startsWith(text)` to `tag.toLowerCase().startsWith(text.toLowerCase())`

If you all or areeba is able to confirm my hunch, feel free to make the changes, test locally and make a PR

**Comment by areeba-dau** [comment]
@dennyabrain You are right! First tag or second, the list shows up only when we type in lowercase.

**Comment by dennyabrain** [comment]
Ok cool so we'll fix it at some point this week. If you see this getting deprioritized, feel free to remind us.

**Comment by duggalsu** [comment]
@dennyabrain I have also confirmed this independently. Capital letters are not working with tags

**Comment by areeba-dau** [comment]
Hi, requesting to add new tags to dashboard. We have updated the existing sheet with the new tags from **rows 4 to 63** (tags above the Yellow line).  https://docs.google.com/spreadsheets/d/1YLS4MyCmGuvMt95rhAeqgZ9hHor5HM8fZ_KqdtNY-F8/edit#gid=321819565

**Comment by tarunima** [comment]
Also need to add four language specific tags:

- lang:hindi
- lang:tamil
- lang:english
- lang:telugu

**Comment by dennyabrain** [comment]
addressed here - [b4898d4](https://github.com/tattle-made/dau-dashboard/commit/b4898d4cf1d81d9cfe73ed09b21e1107f7fda987)

**Comment by dennyabrain** [comment]
lets track the discussion on language tags here - https://github.com/tattle-made/dau-dashboard/issues/15 and close this if needed.
